docs/sources/get-started/labels/non-indexed-labels.md

```diff
@@ -0,0 +1,50 @@
+---
+menuTitle: Non-indexed labels
+title: What are non-indexed labels
+description: Attaching metadata to logs.
+---
+# What are non-indexed labels
+
+One of the powerful features of Loki is parsing logs at query time to extract metadata and build labels out of it.
+However, the parsing of logs at query time comes with a cost which can be significantly high for, as an example,
+large json blobs or a poorly written query using complex regex patterns.
+
+In addition, the data extracted from logs at query time is usually high cardinality, which can’t be stored
+in the index as it would increase the cardinality too much, and therefore reduce the performance of the index.
+
+Non-indexed labels are a way to attach metadata to logs without indexing them. Examples of useful metadata are
+trace IDs, user IDs, and any other label that is often used in queries but has high cardinality and is expensive
+to extract at query time.
+
+## Attaching non-indexed labels to log lines
+
+You have the option to attach non-indexed labels to log lines in the push payload along with each log line and the timestamp.
+For more information on how to push logs to Loki via the HTTP endpoint, refer to the [HTTP API documentation]({{< relref "../../reference/api#push-log-entries-to-loki" >}}).
+
+Alternatively, you can use the Grafana Agent or Promtail to extract and attach non-indexed labels to your log lines.
+See the [Promtail configuration documentation]({{< relref "../../clients/promtail/configuration" >}}) for more information.
```

Comment:
```suggestion
See the [Promtail: Non-indexed labels stage]({{< relref "../../clients/promtail/stages/non-indexed-labels" >}}) for more information.
```

Reply:
i believe we need to point the user to the concreate page with the new stage. it will be more useful for him